website/mails.py

```diff
@@ -150,3 +150,6 @@ def send_mail(to_addr, mail, mimetype='plain', from_addr=None, mailer=None,
 
 DIGEST = Mail('digest', subject='OSF Email Digest')
 TRANSACTIONAL = Mail('transactional', subject='OSF: ${subject}')
+
+PENDING_RETRACTION = Mail('pending_retraction', subject='A registration you administer is pending retraction.')
```

Comment:
@bnosek @saradbowman These are the subject lines for emails sent out to project admins for retraction/embargo approval or disapproval